app/admin/views.py

```diff
@@ -19,28 +22,19 @@
 @admin.route('/<agency_ein>')
 def main(agency_ein=None):
     if not current_user.is_anonymous:
+        if agency_ein is None:
+            agency_ein = current_user.default_agency
```

Comment:
`default_agency` property method should return agency object and `default_agency_ein` should return ein.